backend/social_connections/models.py

```
@@ -143,6 +143,32 @@ def __str__(self):
+    )
+    discord_user_id = models.CharField(max_length=100, db_index=True)
+    discord_username = models.CharField(max_length=100, blank=True)
+    role_id = models.CharField(max_length=100, db_index=True)
+    role_name = models.CharField(max_length=100)
+    total_points = models.PositiveIntegerField()
+    poap_count = models.PositiveIntegerField()
+    assigned_at = models.DateTimeField(auto_now_add=True, db_index=True)
+
+    class Meta:
+        db_table = 'social_connections_discord_earned_role_assignment'
+        ordering = ['-assigned_at']
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)